theatre/studio/src/panels/OutlinePanel/OutlinePanel.tsx

```diff
@@ -15,7 +14,7 @@ const Container = styled.div<{pin: boolean}>`
   position: absolute;
   left: 8px;
   z-index: ${panelZIndexes.outlinePanel};
-  ${pointerEventsAutoInNormalMode};
```

Comment:
Outcome of removing `pointerEventsAutoInNormalMode`:

![Screenflick Movie 115](https://user-images.githubusercontent.com/593118/173887690-6be34a51-a12f-4d9c-a2eb-af09555fcdd6.gif)

Reply:
@AriaMinaei I tested, and it seems the behaviour you are showing (hover styling while dragging) was already present on `main` in the outline items before this PR's changes.

Reply:
If you intended hover styles to be disabled while dragging, I can add a fix for that to this PR.

Reply:
@vezwork oh yeah please do

Reply:
Fixed the hover styles and another issue where useCssCursorLock was not being used when dragging part of the sequence panel @AriaMinaei !